PizzaBuilderProject/PizzaBuilderProject/Program.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PizzaBuilderProject
+{
+    public interface IpizzaBuilder
+    {
+       public  void BuilderDough();
+       public void BuildTopping();
+       public void BuildSauce();
+       public  void Reset();
+    }
+
+    public class whitePizzaBuilder : IpizzaBuilder
+    {
+        private WhitePizza _whitePizza = new WhitePizza();
+        public whitePizzaBuilder()
+        {
+            this.Reset();
+        }
+        public void BuilderDough()
+        {
+            this._whitePizza.Add("white Dough");
+
+        }
+
+        public void BuildSauce()
+        {
+            this._whitePizza.Add("Souce");
+        }
+
+        public void BuildTopping()
```

Comment:
It would be nice if you specified some toppings like (Broccoli, Mushroom, onions) or what you want rather than just writing the word Topping.